Rubrik/Public/New-RubrikLDAPSettings.ps1

```diff
@@ -0,0 +1,94 @@
+﻿#Requires -Version 3
+function New-RubrikLDAPSettings
+{
+  <#  
+      .SYNOPSIS
+      Connects to Rubrik and sets Rubrik cluster settings
+            
+      .DESCRIPTION
+      The New-RubrikSettings cmdlet will set the cluster settings on the system. This does require authentication.
+            
+      .NOTES
+      Adapted by Adam Shuttleworth from scripts by Chris Wahl for community usage
+            
+      .LINK
+      https://github.com/rubrikinc/PowerShell-Module
+            
+      .EXAMPLE
+      New-RubrikSettings -Name "Test LDAP Settings" -baseDN "DC=domain,DC=local" -authServers "192.168.1.8"
+      This will create LDAP settings on the Rubrik cluster defined by Connect-Rubrik function
+  #>
+
+  [CmdletBinding(SupportsShouldProcess = $true,ConfirmImpact = 'High')]
+  Param(
+    # Rubrik server IP or FQDN
+    [String]$Server = $global:RubrikConnection.server,
+    # ID of the Rubrik cluster or me for self
+    [String]$id = '',
+    # API version
+    [ValidateNotNullorEmpty()]
+    [String]$api = $global:RubrikConnection.api,
+    [Parameter(Mandatory=$True)]
+    [string]$name,
+    [Parameter(Mandatory=$True)]
+    [string]$dynamicDNSName,

```

Comment:
I don't believe the variables are case sensitive for the API; you can use the `DynamicDNSName` here to conform to PowerShell standards.

Reply:
I conformed that parameter variable to Powershell standards.